#156222795 Create dashboard for security guards

chadwalt opened the pull request:
#### What does this PR do?
Creates a dashboard for the security guards.
#### Description of Task to be completed?
When the security guard logins into the application, they should be able to see a dashboard which has their name and email address and check serial button
#### How should this be manually tested?
By running the security_dashboard layout.
#### What are the relevant pivotal tracker stories?
[#156222795](https://www.pivotaltracker.com/story/show/156222795)
#### Screenshots
![screenshot_20180329-162252](https://user-images.githubusercontent.com/11385771/38091502-6863eb26-336e-11e8-8a6a-11099f53a758.png)

chadwalt committed:
Add profile image to the dashboard

chadwalt committed:
Add profile image to the dashboard

chadwalt committed:
Add fullname, email, and button

chadwalt committed:
Add fullname, email, and button

muhallan committed:
feat(Security dashboard): modify the layout file

- Add fonts
- Change the margins of views in layouts

muhallan committed:
feat(Security dashboard): modify the layout file

- Add fonts
- Change the margins of views in layouts

chadwalt committed:
Add tool to the dashboard

chadwalt committed:
Add app tools and fix avenir_book.otf duplication

chadwalt committed:
Add id to the toolbar and align it on top of the imageview

chadwalt committed:
Add id to the toolbar and align it on top of the imageview

eric-elem commented on app/src/main/res/values/colors.xml:
```diff
@@ -6,4 +6,5 @@
     <color name="EmailTextAlpha">#B3FFFFFF</color>
     <color name="primaryBalticSea">#1E1D23</color>
     <color name="primaryCharade">#25252f</color>
+    <color name="backgroundDark">#1e1d23</color>
```
This color is already defined on `line 7`.

chadwalt replied:
I have addressed it

eric-elem commented on app/src/main/res/layout/security_dashboard.xml:
```diff
@@ -1,11 +1,69 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/email"
+        android:textColor="#dadcdf"
```
Since this text color `dadcdf` is used more than once (here and in the full_name TextView), it would better to store it's value in colors.xml such that incase have to change it in future, we change it from one place and it reflects everywhere it's used.

chadwalt replied:
Thanks, I have addressed this too.

chadwalt committed:
fix height on the actionbar, set the default color for the text

chadwalt committed:
Merge commit '2919e91' into ft-security-dashboard-156222795

eric-elem commented:
LGTM

muhallan committed:
feat(dashboard activity): add tests for the UI views

suada-haji commented on app/src/main/res/layout/security_dashboard.xml:
```diff
@@ -1,11 +1,69 @@
+        android:layout_width="130dp"
+        android:layout_height="130dp"
+        android:layout_marginTop="12dp"
+        android:src="@mipmap/ic_launcher_round"
```
use `tools` instead of `android`

suada-haji commented on app/src/main/res/layout/security_dashboard.xml:
```diff
@@ -1,11 +1,69 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="13dp"
+        android:text="@string/full_name"
```
Same here

suada-haji commented on app/src/main/res/layout/security_dashboard.xml:
```diff
@@ -1,11 +1,69 @@
+        android:id="@+id/email_address"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/email"
```
Also here

suada-haji commented on app/src/main/res/layout/tool_bar.xml:
```diff
@@ -4,14 +4,13 @@
     android:layout_width="match_parent"
-    android:layout_height="80dp"
+    android:layout_height="?attr/actionBarSize"
     android:minHeight="80dp"
```
place this value in the resource folder

suada-haji commented on app/src/main/res/layout/security_dashboard.xml:
```diff
@@ -1,11 +1,69 @@
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/profile_picture"
+        android:layout_width="130dp"
```
Place all values such as `20dp` in the resource folder

muhallan committed:
feat(review): add all values to resource files

- move all figures to dimens
- move all color codes to colors
- modify tests to work with hidden TextViews

suada-haji reviewed (approved):
Great! Looks good.
